CHANGELOG.md

```diff
@@ -209,3 +209,7 @@
 ## 2024-03-04
 ### bugfix
 - Remove check if the article content is empty from shortcode, which excluded empty articles from the searchmap, preventing them from being added to the database. @CharlRitterDev https://spandigital.atlassian.net/browse/PRSDM-5230
+
+## 2024-03-11
+### Updates
+- Fix any Apple regression stemming from User Dashboard. @CharlRitterDev https://spandigital.atlassian.net/browse/PRSDM-5185
```

Comment:
This should rather say what the actual fix is for this repo i.e. set default font for Presidium to Open Sans and code blocks to monospace and include portal stylesheet which would apply customer specific font overrides